stripper/ze_Mirrors_Edge_Reborn_p3.cfg

```diff
@@ -1,115 +1,232 @@
-;set level 2 immediately after triggering the heli on race level, first place people would constantly suicide to stop the level from advancing
-modify:
-{
-	match:
-	{
-		"classname" "func_button"
-		"targetname" "easybutton"
-	}
-	insert:
-	{
-		"OnPressed" "level1Enable0-1"
-		"OnPressed" "level2Disable0-1"
-		"OnPressed" "LevelCounterSetValueNoFire20-1"
-	}
-}
-
-;Stopmusic fix, music delay.
-modify:
-{
-	match:
-	{
-		"classname" "logic_relay"
-		"targetname" "Lvl1"
-	}
-	delete:
-	{
-		"OnTrigger" "musicadefondoeasyPlaySound01"
-	}
-	insert:
-	{
-		"OnTrigger" "musicadefondoeasy,PlaySound,,3,1"
-	}
-}
-
-modify:
-{
-	match:
-	{
-		"classname" "logic_relay"
-		"targetname" "Lvl2"
-	}
-	delete:
-	{
-		"OnTrigger" "musicadefondoPlaySound0-1"
-	}
-	insert:
-	{
-		"OnTrigger" "musicadefondo,PlaySound,,3,1"
-	}
-}
-
-modify:
-{
-	match:
-	{
-		"classname" "logic_relay"
-		"targetname" "Lvl3"
-	}
-	delete:
-	{
-		"OnTrigger" "musicadefondoPlaySound0-1"
-	}
-	insert:
-	{
-		"OnTrigger" "musicadefondo,PlaySound,,3,1"
-	}
-}
-
-modify:
-{
-	match:
-	{
-		"classname" "logic_relay"
-		"targetname" "Lvl4"
-	}
-	delete:
-	{
-		"OnTrigger" "musicadefondoextremePlaySound0-1"
-	}
-	insert:
-	{
-		"OnTrigger" "musicadefondoextreme,PlaySound,,3,1"
-	}
-}
-
-modify:
-{
-	match:
-	{
-		"classname" "logic_relay"
-		"targetname" "Lvl5"
-	}
-	delete:
-	{
-		"OnTrigger" "musicadefondogodPlaySound0-1"
-	}
-	insert:
-	{
-		"OnTrigger" "musicadefondogod,PlaySound,,3,1"
-	}
-}
-
-;Spawn start fix.
-modify:
-{
-	match:
-	{
-		"classname" "info_player_terrorist"
-		"origin" "144 1312 16"
-	}
-	replace:
-	{
-		"origin" "144 1312 -24"
-	}
```

Comment:
No idea what you did to make the old config show as completely deleted/readded, but please fix if possible.